minitest.gemspec

```diff
@@ -0,0 +1,50 @@
+require_relative 'lib/minitest'
+
+Gem::Specification.new do |spec|
+  spec.name    = "minitest"
+  spec.version = Minitest::VERSION
+
+  spec.homepage = "https://github.com/seattlerb/minitest"
+  spec.authors  = ["Ryan Davis"]
+  spec.email    = ["ryand-ruby@zenspider.com"]
+  spec.licenses = "MIT"
+
+  spec.summary     = "minitest provides a complete suite of testing facilities supporting TDD, BDD, mocking, and benchmarking"
+  spec.description = "minitest provides a complete suite of testing facilities supporting "\
+                     "TDD, BDD, mocking, and benchmarking.\n"\
+                     "minitest/test is a small and incredibly fast unit testing framework. "\
+                     "It provides a rich set of assertions to make your tests clean and readable.\n"\
+                     "minitest/spec is a functionally complete spec engine. It hooks onto "\
+                     "minitest/test and seamlessly bridges test assertions over to spec expectations.\n"\
+                     "minitest/benchmark is an awesome way to assert the performance of your\nalgorithms "\
+                     "in a repeatable manner. Now you can assert that your new co-worker doesn't replace your "\
+                     "linear algorithm with an exponential one!\n" \
+                     "nminitest/mock by Steven Baker, is a beautifully tiny mock (and stub) object framework.\n"\
+                     "minitest/pride shows pride in testing and adds coloring to your test output. "\
+                     "I guess it is an example of how to write IO pipes too."
+
+  spec.require_paths    = ["lib"]
+  spec.files            = `git ls-files`.split("\n")
```

Comment:
Why not put the test files into ``spec.test_files``?
``spec.files = `git ls-files -z`.split("\x0").select { |f| f.match(%r{^(lib)/}) }``